package.json

```diff
@@ -10,7 +10,7 @@
   "license": "MPL-2.0",
   "main": "webpack.config.js",
   "scripts": {
-    "build.dev": "webpack --mode development --display-error-details --progress --verbose --env.dev",
+    "build.dev": "webpack --mode development --display-error-details --progress --env.dev",
```

Comment:
how about adding `--verbose` to `build.watch`?

Reply:
sure sounds good. I'll add it 👍